docs_src/getting-started/Ch-GettingStartedDevelopers.md

```diff
@@ -2,82 +2,92 @@
-Foundry consists of a collection of microservices and SDK tools. The
-microservices and SDKs are mostly written in Go or C with some legacy
-services written in Java (EdgeX was originally written in Java). These
-documentation pages provide a developer with the information and
+EdgeX is a collection of more than a dozen micro services that are deployed to provide a minimal edge platform capability. 
+EdgeX consists of a collection of reference implementation services and SDK tools. The micro services and SDKs are written in Go or C. 
+These documentation pages provide a developer with the information and
 instructions to get and run EdgeX Foundry in development mode - that is
 running natively outside of containers and with the intent of adding to
 or changing the existing code base.
 
 ## What You Need
 
-**Hardware**
+### Hardware
 
-EdgeX Foundry is an operating system (OS)-agnostic and hardware
-(HW)-agnostic edge software platform. Minimum platform requirements are
-being established. At this time use the following recommended
-characteristics:
+EdgeX Foundry is an operating system (OS) and hardware (HW)-agnostic edge software platform. 
+See the reference page for [platform requirements](./quick-start/index.md#reference-platform-requirements). These provide guidance on a minimal platform to run the EdgeX platform.  However, as a developer, you may find that additional memory, disk space, and improved CPU are essential to building and debugging.
 
--   Memory: minimum of 1 GB
--   Hard drive space: minimum of 3 GB of space to run the EdgeX Foundry
-    containers, but you may want more depending on how long sensor and
-    device data is retained
--   OS: EdgeX Foundry has been run successfully on many systems including, but not limited to the following systems
-    -   Windows (ver 7 - 10)
-    -   Ubuntu Desktop (ver 14-16)
-    -   Ubuntu Server (ver 14)
-    -   Ubuntu Core (ver 16)
-    -   Mac OS X 10
+### Software
 
-**Software**
+Developers need to install the following software to get,
+run and develop EdgeX Foundry micro services:
 
-Developers will need to install the following software in order to get,
-run and develop EdgeX Foundry microservices:
-
-**git** - a free and open source version control (SVC) system used to
+#### Git
+Use this free and open source version control (SVC) system to
 download (and upload) the EdgeX Foundry source code from the project's
-GitHub repository. See <https://git-scm.com/downloads> for download and
+GitHub repositories. See <https://git-scm.com/downloads> for download and
 install instructions. Alternative tools (Easy Git for example) could be
 used, but this document assumes use of git and leaves how to use
 alternative SVC tools to the reader.
 
-**MongoDB** - by default, EdgeX Foundry uses MongoDB (version 4.2 as of
-this writing) as the persistence mechanism for sensor data as well as
-metadata about the devices/sensors that are connected. See
+#### Redis
```

Comment:
do we wanna mention anything about the Redis Desktop Manager for managing the DB?